os-eap7-launch/added/launch/activemq-subsystem.xml

```diff
@@ -15,9 +14,6 @@
             <param name="direct-deliver" value="false"/>
         </http-acceptor>
         <in-vm-acceptor name="in-vm" server-id="0"/>
-        <broadcast-group name="bg-group1" connectors="http-connector" jgroups-channel="ee"/>
```

Comment:
@jmesnil -- This is just me thinking out loud for the record.

If we leave this line inI believe this leaves the broker as discoverable by remote clients (say in an app w/ 1 messaging server pod and n > 1 front end pods that communicate with the messaging server.

BUT, leaving this in place also means if people deploy the front end onto the messaging server and scale that up, which seems a more likely scenario, then we get the problem of clients sending messages to random unrelated servers.

So, we shouldn't leave it. And that's consistent with the currently approved doc verbiage "ActiveMQ Artemis is only supported for messaging within a single pod with JBoss EAP instances, and is only enabled when JBoss AMQ is absent."

Reply:
@bstansberry you are correct. broadcast-group (and discovery-group and cluster-connections) must be removed to ensure that this node will not advertise itself or connect to other nodes in any fashion.

This will isolate it from any other messaging servers

Reply:
Removing HA support for the messaging subsystem should also require changes to the `RemoteConnectionFactory` so that is is configured as:

```
<connection-factory name="RemoteConnectionFactory" connectors="http-connector" entries="java:jboss/exported/jms/RemoteConnectionFactory" reconnect-attempsts="-1" />
```

This uses the default value for the attributes:
* `ha` (which is `false` as we have no other servers to failover)
* `block-on-acknowledge` (which is `false`). Blocking on acknowledgement is required or transparent failover. Since there are no other servers to failover, there is no good reason to keep blocking.

Reply:
Thanks Jeff, I'll check and update this too.